### PR TITLE
lint: enable stylecheck

### PR DESCRIPTION
**Pull request by zirain**

### CODE REVIEW

**Review by arkodg — APPROVED** [review]
LGTM thanks !

File: internal/provider/kubernetes/controller_test.go [hunk 1]
```diff
@@ -318,7 +317,7 @@ func TestEnqueueManagedClass(t *testing.T) {
 		tc := testCases[i]
 
 		// Create the reconciler.
-		logger := logging.DefaultLogger(v1alpha1.LogLevelInfo)
+		logger := logging.DefaultLogger(egcfgv1a1.LogLevelInfo)
```
Comment (AliceProxy) [inline internal/provider/kubernetes/controller_test.go:320]: Is this a manual change, or does this linter help ensure that named imports share the same name across files?

**Review by AliceProxy — APPROVED** [review]

**Review by qicz — APPROVED** [review]
LGTM

File: internal/provider/kubernetes/controller_test.go (hunk 1 above)
Comment (zirain) [reply internal/provider/kubernetes/controller_test.go:320]: no, just ensure you cannot import same package more than once.

**Pull request merged**